tests/functionnal/Appliance_Item_Relation.php

```diff
@@ -85,7 +85,7 @@ public function testCountForApplianceItem() {
       $this->integer(\Appliance_Item_Relation::countForMainItem($appitem))->isIdenticalTo(0);
 
       $this->login();
-      $this->setEntity(0, true); //locations are in root entity not recursive
+      $this->setEntity(__('Root entity'), true); //locations are in root entity not recursive
```

Comment:
I'm really not sure for that one. Why a simple "0" is an issue?

Reply:
First parameter of this method is the entity name. Result was a switch to entity `false`, with this SQL warning: ``SELECT `id`, `ancestors_cache`, `entities_id` FROM `glpi_entities` WHERE `id` IN ('')``.

Reply:
Maybe `setEntity` could be updated so it can take an integer (if not too complex of course)? Most of the time, we'll use `0` if we need the root entity, this is more "complicated" to write a localized string.

Reply:
done

Reply:
:+1: